templateChecks.sh

```diff
@@ -25,3 +25,6 @@ pushd template/
 ./gradlew compileAllVersionsStagingDebugSources
 ./gradlew compileAllVersionsStagingDebugAndroidTestSources
 ./gradlew compileAllVersionsStagingDebugUnitTestSources
+
+./gradlew test
+./gradlew lint
```

Comment:
And maybe in the future we should run this not only for the template but for the whole project.